<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="textbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sum">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_minus">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_result">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="number9">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="number7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="number8">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_multiplication">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_division">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="number4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="number5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="number6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="number1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="number2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="number3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cls">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>cls</string>
   </property>
  </widget>
  <widget class="QPushButton" name="number0">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_percent">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>-/+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_decimal">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sin">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>sin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cos">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>cos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_tan">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>tan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cot">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>cot</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_log">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>log</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sqrt">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>sqrt</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
